// @chakra-ui/theme/dist/chunk-SPKLG64B.mjs
import {
  stepperTheme
} from "./chunk-HAMNZXTQ.mjs";
import {
  switchTheme
} from "./chunk-KGFPXNP4.mjs";
import {
  tableTheme
} from "./chunk-WHSEAAWU.mjs";
import {
  tabsTheme
} from "./chunk-VJDRZQAM.mjs";
import {
  tagTheme
} from "./chunk-XJB4FHJM.mjs";
import {
  textareaTheme
} from "./chunk-LHPMQNXN.mjs";
import {
  tooltipTheme
} from "./chunk-6Y3UQESG.mjs";
import {
  progressTheme
} from "./chunk-ZEWNBRBK.mjs";
import {
  radioTheme
} from "./chunk-JDPWIQ7H.mjs";
import {
  selectTheme
} from "./chunk-46I2TFNY.mjs";
import {
  skeletonTheme
} from "./chunk-YSULOCDF.mjs";
import {
  skipLinkTheme
} from "./chunk-RSEFU3LC.mjs";
import {
  sliderTheme
} from "./chunk-IBL46MNN.mjs";
import {
  spinnerTheme
} from "./chunk-Y6RR3LE6.mjs";
import {
  statTheme
} from "./chunk-EY6ACYQS.mjs";
import {
  kbdTheme
} from "./chunk-VQ2DUS6J.mjs";
import {
  linkTheme
} from "./chunk-2T5TO72C.mjs";
import {
  listTheme
} from "./chunk-AQJ6AIMN.mjs";
import {
  menuTheme
} from "./chunk-FIWRYWVW.mjs";
import {
  modalTheme
} from "./chunk-XHTV2ZQR.mjs";
import {
  numberInputTheme
} from "./chunk-GDLDHFM2.mjs";
import {
  pinInputTheme
} from "./chunk-RFSQEQHG.mjs";
import {
  popoverTheme
} from "./chunk-YJYMWHG4.mjs";
import {
  drawerTheme
} from "./chunk-PQSF5PVS.mjs";
import {
  editableTheme
} from "./chunk-YZVOF4MP.mjs";
import {
  formTheme
} from "./chunk-RUW3STUO.mjs";
import {
  formErrorTheme
} from "./chunk-D55YRTLZ.mjs";
import {
  formLabelTheme
} from "./chunk-OYCZTG3L.mjs";
import {
  headingTheme
} from "./chunk-X4O3JXYB.mjs";
import {
  inputTheme
} from "./chunk-VB5QFFP5.mjs";
import {
  breadcrumbTheme
} from "./chunk-OZJFUHHJ.mjs";
import {
  buttonTheme
} from "./chunk-3WZBHGUV.mjs";
import {
  cardTheme
} from "./chunk-PZHBQTKY.mjs";
import {
  checkboxTheme
} from "./chunk-3XG4KRSA.mjs";
import {
  closeButtonTheme
} from "./chunk-C4P5PUHY.mjs";
import {
  codeTheme
} from "./chunk-CAKKPKRW.mjs";
import {
  containerTheme
} from "./chunk-NDGGJV23.mjs";
import {
  dividerTheme
} from "./chunk-5Z4IVOU3.mjs";
import {
  accordionTheme
} from "./chunk-T276HDLM.mjs";
import {
  alertTheme
} from "./chunk-YOXAUO7A.mjs";
import {
  avatarTheme
} from "./chunk-BNDUAR5A.mjs";
import {
  badgeTheme
} from "./chunk-IBJP6BGU.mjs";

// src/components/index.ts
var components = {
  Accordion: accordionTheme,
  Alert: alertTheme,
  Avatar: avatarTheme,
  Badge: badgeTheme,
  Breadcrumb: breadcrumbTheme,
  Button: buttonTheme,
  Checkbox: checkboxTheme,
  CloseButton: closeButtonTheme,
  Code: codeTheme,
  Container: containerTheme,
  Divider: dividerTheme,
  Drawer: drawerTheme,
  Editable: editableTheme,
  Form: formTheme,
  FormError: formErrorTheme,
  FormLabel: formLabelTheme,
  Heading: headingTheme,
  Input: inputTheme,
  Kbd: kbdTheme,
  Link: linkTheme,
  List: listTheme,
  Menu: menuTheme,
  Modal: modalTheme,
  NumberInput: numberInputTheme,
  PinInput: pinInputTheme,
  Popover: popoverTheme,
  Progress: progressTheme,
  Radio: radioTheme,
  Select: selectTheme,
  Skeleton: skeletonTheme,
  SkipLink: skipLinkTheme,
  Slider: sliderTheme,
  Spinner: spinnerTheme,
  Stat: statTheme,
  Switch: switchTheme,
  Table: tableTheme,
  Tabs: tabsTheme,
  Tag: tagTheme,
  Textarea: textareaTheme,
  Tooltip: tooltipTheme,
  Card: cardTheme,
  Stepper: stepperTheme
};

export {
  components
};
